src/sentry/onboarding_tasks/backends/organization_onboarding_task.py

```diff
@@ -0,0 +1,44 @@
+from django.db import IntegrityError, transaction
+from django.db.models import Q
+from django.utils import timezone
+
+from sentry.models import OnboardingTaskStatus, OrganizationOnboardingTask, OrganizationOption
+from sentry.onboarding_tasks.base import OnboardingTaskBackend
+
+
+class OrganizationOnboardingTaskBackend(OnboardingTaskBackend):
+    MODEL = OrganizationOnboardingTask
```

Comment:
Nit: I feel like we usually use all caps for constants, not class variables

Reply:
@wedamija Yea I can change this